crates/holochain_zome_types/src/zome.rs

```diff
@@ -27,15 +27,19 @@ use std::sync::Arc;
 #[derive(Serialize, Deserialize, Hash, Clone, Debug, PartialEq, Eq, PartialOrd, Ord)]
 #[cfg_attr(feature = "full-dna-def", derive(shrinkwraprs::Shrinkwrap))]
 #[cfg_attr(feature = "test_utils", derive(arbitrary::Arbitrary))]
-pub struct Zome {
+pub struct Zome<T = ZomeDef> {
```

Comment:
Would rather not even have this default, I hope we move away from zome agnosticism, where we always know whether we're referring to an integrity or coordinator zome.